### ISSUE TITLE
Iteration/Comment pair doesn't effect Right Hand Side conversation

### ISSUE THREAD

**Issue by kotp** [exercism/exercism#5635]
![comments_go_with_which_iteration](https://user-images.githubusercontent.com/16747/131736790-1868d753-dc8f-434e-8435-a04a6f75759b.png)

If clicking on the iteration, I don't think it matters if the right hand side comment pane scrolls to the beginning of the iteration, but if I click on the number above the iteration, that was the intuitive thing that was expected.

Worse, though, if I am on iteration 2 out of 5, and am literally clicked on iteration 2, and make a new comment, it goes as an entry on iteration 5, according to the number indicators (and it is indeed in the last comment slot on the right hand side).

**Comment by kotp** [comment]
The link button, @joshgoebel ?

**Comment by joshgoebel** [comment]
<img width="40" alt="Screen Shot 2021-09-01 at 4 27 14 PM" src="https://user-images.githubusercontent.com/6473/131740232-812894b4-187f-4388-8850-ddc1354ae6ca.png">

**Comment by kotp** [comment]
Yep.  I have.   It works both ways.  The additional thing, then is explicitly clicking on the number above the iteration to go to that "start" but also, when commenting on iteration 2 out of 5, and referring to iteration 2 specifically, there should be a way to do so (without having to state "about iteration 2… I have a comment."

**Comment by iHiD** [comment]
All replies now go at the end of a conversation so being able to attach them to old iterations won't work with the UI as it is now (as opposed to how it was in v2). So I'm going to close this.
